src/routes/auth.router.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express');
+const authController = require('../controllers/auth.controller');
+const { catchError } = require('../utils/catchError');
+const { noAuthMiddleware } = require('../middlewares/noAuth.middleware');
+const { authMiddleware } = require('../middlewares/auth.middleware');
+
+const authRouter = new express.Router();
+
+authRouter.post(
+  '/registration',
+  noAuthMiddleware,
+  catchError(authController.register),
+);
+
+authRouter.get(
+  '/activation/:activationToken',
+  noAuthMiddleware,
+  catchError(authController.activate),
+);
+
+authRouter.post('/login', noAuthMiddleware, catchError(authController.login));
+
+authRouter.get(
+  '/refresh',
+  noAuthMiddleware,
```

Comment:
The use of 'noAuthMiddleware' on the '/refresh' route might be incorrect. Typically, a refresh token is used when a user is already authenticated, so 'authMiddleware' might be more appropriate here instead of 'noAuthMiddleware'.